src/main/java/org/opentripplanner/routing/algorithm/raptor/transit/TripPatternForDate.java

```diff
@@ -11,10 +11,20 @@
  * having to do any filtering by date during the search itself.
  */
 public class TripPatternForDate implements TripPatternInfo<TripSchedule> {
```

Comment:
That makes sense. I did find the name TripPatternInfo a little confusing, but I just considered all names would potentially change since this was a relatively new system. We can rename this interface.